game/static/game/css/board.css

```
@@ -846,35 +1194,44 @@ body {
+    letter-spacing: -0.1px;
     font-weight: 700;
     opacity: 0.7;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    min-width: 0;
+    flex: 1 1 auto;
 }
 
 .clock .time {
     font-family: 'Consolas', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)